RNTester/Podfile

```diff
@@ -98,8 +98,38 @@ target 'RNTester-macOSIntegrationTests' do
   pod 'React-RCTTest', :path => "RCTTest"
 end
 
+# [TODO(macOS GH#217): these are special targets used by the internal Microsoft build pipeline
```

Comment:
The TODO macOS is misleading here. A specific comment about this diff may make it easier to maintain in the future

Reply:
Similar elsewhere in the review

Reply:
Ya, you're right.  My intention was to mark deltas from upstream, but we do that with an ADO bug #, not this GH issue #.   I'll make another change to update all the podspec comments.